packaging/src/docker/README.md

```diff
@@ -59,7 +60,7 @@ If the version is not provided, it will read the version from current `pom.xml`:
 
 For example, the following command uses Hive 3.1.3, Hadoop `hadoop.version` and Tez `tez.version` to build the image,
```

Comment:
Change `uses Hive 3.1.3` to `Hive 4.0.0`?

Reply:
Done